Content.Client/Guidebook/Controls/GuideReagentEmbed.xaml.cs

```diff
@@ -204,6 +210,25 @@ private void GenerateControl(ReagentPrototype reagent)
         description.PushNewline();
         description.AddMarkupOrThrow(Loc.GetString("guidebook-reagent-physical-description",
             ("description", reagent.LocalizedPhysicalDescription)));
+
+        if (_config.GetCVar(CCVars.ContrabandExamine))
+        {
+            // Department-restricted text
+            if (reagent.AllowedJobs.Count > 0 || reagent.AllowedDepartments.Count > 0)
+            {
+                description.PushNewline();
+                description.AddMarkupPermissive(
+                    _contraband.GenerateDepartmentExamineMessage(reagent.AllowedDepartments, reagent.AllowedJobs, ContrabandItemType.Reagent));
+            }
+            // Other contraband text
+            else if (reagent.ContrabandSeverity != null &&
+                     _prototype.TryIndex(reagent.ContrabandSeverity.Value, out var severity))
```

Comment:
```suggestion
                     _prototype.Resolve(reagent.ContrabandSeverity.Value, out var severity))
```
This should log an error instead of silently failing.
See https://github.com/space-wizards/space-station-14/issues/39115